### DISCUSSION TITLE
Add a button to VIEW the current page from the backed.

### DISCUSSION THREAD

**Discussion by atheimer** [GitbookIO/community#564] [Feature Requests]
I edit a page.. then I want to view the public version of it...

**Comment by taszap** [comment]
Hi @atheimer 

This is already possible in the change request.  Click on the button 'Preview' and you will see how the changes will appear on public site

![CleanShot 2024-04-09 at 13 46 54](https://github.com/GitbookIO/community/assets/63643069/8f4ee84f-8615-40e7-9732-4ccfb1215f04)
